processing/data_collection/gazette/spiders/rj_campos_goytacazes.py

```diff
@@ -0,0 +1,45 @@
+import re
+import dateparser
+
+from datetime import datetime
+from scrapy import Request
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+class RjCampoGoytacazesSpider(BaseGazetteSpider):
+    GAZETTE_ELEMENT_CSS = 'ul.ul-licitacoes li'
+    MUNICIPALITY_ID = '3301009'
```

Comment:
The ```MUNICIPALITY_ID``` property was updated to ```TERRITORY_ID```

Reply:
Addressed at https://github.com/okfn-brasil/querido-diario/pull/235/commits/93e8eefa39fc53803450fca9d97144fcc1fa6949.

Reply:
@chamun you need to update this branch with the commit.